### ISSUE TITLE
Minimal Implementation Walk Through

### ISSUE THREAD

**Issue by kpeeples** [GeorgiaCyber/kinetic#1]
@bitskri3g  I like the goal of Kinetic which is to have a general baseline and a site specific overlay/pillar.  I will be glad to help build a blog/tutorial/etc on a Minimal Implementation that can be posted as part of the readme or a link to include in the readme.   I think that will help provide a quick start guide for easy adoption.  Let me know your thoughts and how I can help.

**Comment by bitskri3g** [comment]
@kpeeples yes that's the goal.  I need to port more code from the old vta piece for it to be viable.  Give me a few more weeks.

**Comment by bitskri3g** [comment]
@kpeeples We're almost at the point where we can apply a 1.0 tag.  Since you don't know the new codebase, I'm curious to see how well you're able to navigate it.  Some stuff should look familiar, some stuff not.  If you want to continue with README.md and see what you can glean from the source, by all means.

Our environment-specific pillar for prod is at https://git.cybbh.space/vta/kinetic-pillar.
